### PR TITLE
NT-1765:Refactor ActivityFeedActivity

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What

Remove butter knife from ActivityFeedActivity and replaced with view binding

# 🤔 Why

deprecated butterknife

# 🛠 How

1- change ActivityFeedActivity from java to kotlin
3-remove butter-knife and replace it with viewbinding

# 👀 See

https://user-images.githubusercontent.com/1075310/107690947-80284280-6cb3-11eb-80d3-079fea83c9d9.mp4

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# 📋 QA

1-Login with Creator account
2-Check activity from the 
left side menu

# Story 📖

https://kickstarter.atlassian.net/browse/NT-1765

### CODE REVIEW

**Review by Arkariang — APPROVED** [review]
lgtm, just the heads up, that in this branch also has the tanksactivity code, it will be conflicts

**Comment by Arkariang** [comment]
Tested on my environment, looks good. Just a heads up, you have in this branch the changes for the ThanksActvity you'll have some conflicts once the other is merged

File: app/src/main/java/com/kickstarter/ui/activities/ActivityFeedActivity.kt [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        super.onCreate(savedInstanceState)
+        binding = ActivityFeedLayoutBinding.inflate(layoutInflater)
+        val view: View = binding.root
+        setContentView(view)
+
+        currentUser = environment().currentUser()
+        adapter = ActivityFeedAdapter(viewModel.inputs)
+        binding.recyclerView.adapter = adapter
+        binding.recyclerView.layoutManager = LinearLayoutManager(this@ActivityFeedActivity)
+
```
Comment (leighdouglas) [inline app/src/main/java/com/kickstarter/ui/activities/ActivityFeedActivity.kt:37]: nit: Add whitespace to separate these codeblocks and subscriptions to improve readability

**Review by leighdouglas — APPROVED** [review]
Looks good to me! Is the `ThanksActivity` class supposed to be apart of this PR?

**Pull request merged**